WORKFLOW.rst

```diff
@@ -0,0 +1,216 @@
+====================================
+ RPC OpenStack Development Workflow
+====================================
+
+Summary
+=======
+
+It is currently necessary, to change the way the RPC OpenStack team develops
+the project (i.e., adds features, fixes bugs). The old workflow is described
+in the :ref:`Old Workflow <old-workflow>` section.
+
+The new workflow is outlined under :ref:`Proposed Workflow
+<proposed-workflow>`
+
+.. _old-workflow:
+
+RPC OpenStack's Old Workflow
+============================
+
+Facts
+-----
+
+#. |rpcos| is configured with the following branches:
+
+   - ``master``
+
+   - ``kilo``
+
+   When we decide to support liberty, a third branch will appear - ``liberty``.
+
+#. Bug fixes move from ``master`` to the other branch(es) via cherry-picks.
+
+#. Bugs are hand-managed by developers and backports are tracked haphazardly.
```

Comment:
A perfect example of what I'm talking about here is #525. This could have been closed a long time ago, but because of our existing workflow, we only just closed it seconds ago.